peer_mentoring/account_management/templates/profile_index.html

```diff
@@ -13,7 +13,7 @@
       {% if profile.user != request.user %}
       {% if profile.user not in request.user.userprofile.friends.all %}
       <div>
-        <form method="post" id="friend_request" action="{% url 'account_management:profile_home' %}" style="display:inline;">
+        <form method="post" id="friend_request" action="{% url 'account_management:send_request' user.id %}" style="display:inline;">
```

Comment:
Was this the regression? Why this change?

Reply:
I made this change because the send request view should send the friend request. However, I initially had the profile home url cause i copied the url code within an href from a different file instead of rewriting and forgot to change.